src/pages/Homepage/Trendingitem/Trendingitem.jsx

```diff
@@ -1,7 +1,47 @@
 import React from "react";
-
+import "./Trendingitem.css";
+import Pasta from "../../../assets/pasta.jpg";
+import Burger from "../../../assets/burger.jpg";
+import Pizza from "../../../assets/pizza.png";
+import Pancake from "../../../assets/pancake.jpg";
+import Salad from "../../../assets/salad.jpg";
+import { Card } from "../../../components/Card/Card";
 const Trendingitem = () => {
-  return <div>Trending Item</div>;
+  const trendingItemList = [
```

Comment:
I think its an list component so naming it like TrendingItemsList or something similar will be better